curriculum/challenges/english/16-the-odin-project/top-learn-block-and-inline/learn-block-and-inline-question-g.md

```diff
@@ -0,0 +1,35 @@
+---
+id: 65704482e7b02272663824e5
+title: Learn Block and Inline Question G
+challengeType: 15
+dashedName: learn-block-and-inline-question-g
+---
+# --description--
+
+Now that you have some basic understanding of `block` and inline `elements`, pull up your list of `block` and `inline` elements from a couple questions ago and see if you can identify which of the following elements are `block` elements and which are `inline` elements.
+
+# --question--    
+
+## --text--
+
+Which of the following is NOT an `inline` element?
+
+## --answers--
+
+`form` element
+
+---
+
+`anchor` element
+
+---
+
+image element
+
+---
+
+`span` element
```

Comment:
This was meant to be an "either-or". Two options:

- Wrap them all in code tags, refer to "image element" as `img` element.
- Wrap none in code tags, leave text as it is now.

Reply:
shit, something went wrong when committing. I understood what you meant.